src/components/SearchNetwork/SearchNetwork.jsx

```diff
@@ -0,0 +1,142 @@
+// View of all messages between two specific users
+// Reached by path '/search'
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+
+//Import Custom Components
+
+import SearchBar from './SearchBar';
+import SearchResults from './SearchResults';
+
+// MATERIAL UI
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import {
+  Typography,
+  Select,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  FormHelperText,
+} from '@material-ui/core';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import { spacing } from '@material-ui/system';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    '& > *': {
+      margin: theme.spacing(1),
+    },
+  },
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 120,
+    maxWidth: 300,
+  },
+}));
+
+function SearchNetwork() {
+  const dispatch = useDispatch();
+  const classes = useStyles();
+
+  // Grab information from Global Redux Store
+  const features = useSelector((store) => store.features);
+  const service = useSelector((store) => store.vendorTypes);
+
+  // Local state that captures input data
+  const [specialFeatures, setSpecialFeatures] = useState('');
+  const [vendorTypes, setVendorTypes] = useState('');
+
+  const handleVendorChange = (evt) => {
+    // let newUser = evt.target.value;
+    // if (newUser === '-1'){
+    //   history.push(`/discover`);}
+    // else {
+    // userChange(newUser);
+    // }
+  };
+
+  const handleCategoryChange = (evt) => {
+    // let newUser = evt.target.value;
```

Comment:
Don't be afraid to delete unused code! That's what we've got git for, in case we need it back later.

PR Reviewers -- this is something you should be calling out, when you review each other's code